src/data/resolvers/events.ts

```diff
@@ -58,12 +58,10 @@ export const eventsResolvers = ({
         )) as ColonyClientV6;
 
         const colonyInRecoveryMode = await colonyClient.isInRecoveryMode();
+        const enterRecoveryEvents =
+          recoveryStartedEvents?.data?.recoveryAllEnteredEvents || [];
 
-        const enterRecoveryEvents = await getLogs(
-          colonyClient,
-          colonyClient.filters.RecoveryModeEntered(null),
-        );
-        const [mostRecentRecoveryStarted] = enterRecoveryEvents.reverse();
+        const mostRecentRecoveryStarted = [...enterRecoveryEvents].reverse()[0];
```

Comment:
Please keep the original destructured code style, as it's much easier to read.

```suggestion
        const [mostRecentRecoveryStarted] = [...enterRecoveryEvents].reverse();
```

Reply:
@rdig but it wasnt working

Reply:
Then something else is wrong here, since those lines are syntactically the same, I just asked you to make that change to more easily readable.

You can even test this:
```javascript
const array = [1, 2, 3];

const test1 = [...array].reverse()[0];
const [test2] = [...array].reverse();

console.log(test1 === test2); // true
```

Reply:
let me retest it, i remember that i was fighting with weird error :D but will recheck it

Reply:
ok, i was wrong, updated